tokio/src/runtime/builder.rs

```diff
@@ -67,7 +67,7 @@ pub struct Builder {
     pub(super) before_stop: Option<Callback>,
 }
 
-pub(crate) type ThreadNameFn = Arc<dyn Fn() -> String + Send + Sync + 'static>;
+pub(crate) type ThreadNameFn = std::sync::Arc<dyn Fn() -> String + Send + Sync + 'static>;
```

Comment:
Yes, because loom arc's don't accept `T: ?Sized`, so we need to use the std version.

Reply:
oh, gotcha. would be nice to fix this upstream...but i also think we don't need `loom` leak-checking for this, since we don't do anything untoward with it, so the tests are probably a little faster anyway if `loom` doesn't have to think about this one.

Reply:
Not possible without `CoerecedUnsized` being stable https://github.com/tokio-rs/loom/issues/147#issuecomment-628318974

Reply:
ah, yup, carry on!